percona/controller/pgcluster/controller.go

```diff
@@ -524,15 +541,64 @@ func (r *PGClusterReconciler) handleMonitorUserPassChange(ctx context.Context, c
 	return nil
 }
 
-func (r *PGClusterReconciler) reconcileCustomExtensions(cr *v2.PerconaPGCluster) {
+func (r *PGClusterReconciler) reconcileCustomExtensions(ctx context.Context, cr *v2.PerconaPGCluster, postgresCluster *v1beta1.PostgresCluster) error {
```

Comment:
I don't see you use `postgresCluster` anywhere, do you need it?

Reply:
no, sorry! I need to delete it.